index.html

```diff
@@ -350,10 +350,18 @@ <h2 style="text-align:right;">Contributed by</h2>
     </p>
   </footer>
 
+  <script src="./js/libs/jsonld/jsonld.min.js"></script>
   <script>
-    initFieldsData();
-    initCallbacks();
-    loadStateFromStorage();
+    loadSpdxData().then(licenses => {
+      SPDX_LICENSES = licenses;
+      SPDX_LICENSE_IDS = licenses.map(license => license['licenseId']);
+      return loadContextData();
+    }).then(contexts => {
+      initJsonldLoader(contexts);
+      initFields();
+      initCallbacks();
+      loadStateFromStorage();
+    });
```

Comment:
```suggestion
    Promise.all(([loadSpdxData(), loadContextData()]).then(results => {
      const [licenses, contexts] = results;
      SPDX_LICENSES = licenses;
      SPDX_LICENSE_IDS = licenses.map(license => license['licenseId']);

      initJsonldLoader(contexts);
      initFields();
      initCallbacks();
      loadStateFromStorage();
    });
```

You can load both the SPDX licenses and the context in parallel, instead of sequentially. This should avoid increasing load time for users with high latency.